arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/raise/NullableSpec.kt

```diff
@@ -69,6 +69,22 @@ class NullableSpec : StringSpec({
     } shouldBe "1"
   }
 
+  "binding either in nullable" {
+    nullable {
+      val number = Either.Right("s".length)
+      val string = number.map(Int::toString).bind()
```

Comment:
I'm more a fan of `ignoreErrors` as @kyay10 proposed on Slack, but how is this working in this PR? 🤔

Reply:
By pure chance: `.bind()` in `nullable` works over `Nothing?`, and `Either.Right` constructs `Either<Nothing, A>` (so in some sense "there's no error to be ignored").

Reply:
Oh, _Right_. I missed that that was inferring to `Either<Nothing, Right>`.